torch/csrc/autograd/custom_function.cpp

```diff
@@ -363,7 +363,9 @@ static optional_variable_list _process_backward_mode_ad(
   int num_diff_outputs = 0;
 
   for (const auto i : c10::irange(num_outputs)) {
-    // For outputs that are not tensors, put a placeholder undefined input.
+    // For outputs that are not tensors put a placeholder undefined input.
+    // Note that below, we also put a placeholder undefined inputs for when
```

Comment:
Nit: merge the two sentences: we put undefined for non-Tensor and Tensors that are not differentiable